libraries/AP_AHRS/LogStructure.h

```diff
@@ -108,7 +108,7 @@ struct PACKED log_POS {
 };
 
 // @LoggerMessage: RATE
-// @Description: Desired and achieved vehicle attitude rates
+// @Description: Desired and achieved vehicle attitude rates in VTOL/Copter/Heli modes
```

Comment:
```suggestion
// @Description: Desired and achieved vehicle attitude rates.  Logs VTOL controller information for Planes.
```

This will also get out of date if Blimp writes them.

You were missing Sub....

Reply:
changed to an exclusion instead of inclusion